11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,65 @@
+---
+title: blacklist whitelist
+slug: blacklist-whitelist
+flag:
+  level: avoid
+  text: racist term
+defined: true
+excerpt: 'umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include;" carries anti-Black sentiments'
+speech: noun; verb
+alt_words:
+  blacklist (noun):
+    - blocklist
+    - denylist
+    - excludelist
+  whitelist (noun):
+    - allowlist
+    - passlist
+    - trustlist
+    - safelist
+  to blacklist (verb):
+    - to block
+    - to deny
+    - to exclude
+  to whitelist (verb):
+    - to allow
+    - to trust
+    - to include
+reading:
+  - text: UK security agency bans blacklist and whitelist for being racist
+    href: https://www.techspot.com/news/85080-uk-cyber-security-agency-stop-using-racist-blacklist.html
+  - text: Google Chrome team moves away from black/whitelist to be more inclusing
+    href: https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/
+  - text: Google developers styleguide entry for blacklist
+    href: https://developers.google.com/style/word-list#blacklist
+  - text: '"Blacklists” and “whitelists": a salutary warning concerning the prevalence of racist language in discussions of predatory publishing'
+    href: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/
+---
+
+colloquially: umbrella term used to represent a list of entities/people who are to be allowed or denied access to a service, privilege, or recognition: to "blacklist" is "to exclude", to "whitelist" means "to include;" carries anti-Black sentiments
+
+## Issues
+
+<!-- Black/white admittance and denial (Jim Crow) -->
+
+The first problem is the admittance and denial of people based on their race and skin color. The terms are rooted in racist history: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```

Comment:
```suggestion
The first problem is the admittance and denial of people based on their race and skin color. The terms are rooted in racist history: "blacklist" started appearing when the enslavement and deportation of Africans by Europeans began.
```
I think we'll want to remove/adjust the above section, as to me it feels in direct contradiction with line 53?

Reply:
You're right. I've moved things around a little: I kept the reference to the 1610 definition because I think the correlation between the peak of the transatlantic slave trade and the term being coined in a dictionary is important, but I fixed the chronological inconsistency.